active-directory-java-native-headless-v2/pom.xml

```diff
@@ -0,0 +1,74 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>public-client-msal4j-sample</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0</version>
+    <name>public-client-msal4j-sample</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>msal4j</artifactId>
+            <version>0.2.0-preview</version>
```

Comment:
We should update to use the latest msal4j version. 0.7.0-preview.

Reply:
Sure

Reply:
I think API changed in the new version. So we need to modify code accordingly.